src/main/java/nl/nn/testtool/Checkpoint.java

```diff
@@ -104,6 +106,14 @@ public Checkpoint(Report report, String threadName, String sourceClassName,	Stri
 		this.level = level;
 	}
 
+	public Span getSpan() {
+		return span;
+	}
+
+	public void setSpan(Span span) {
```

Comment:
Can you swap places of the set and get method (with all other set/get methods in this file the set comes before the get)?
Can you move the set and get method to the end of the file to keep the order of all the set/get methods in the file in sync with the order of the corresponding vars?

Reply:
Added the change